src/main/java/com/yelp/nrtsearch/tools/nrt_utils/state/StateCommandUtils.java

```diff
@@ -95,7 +95,7 @@ public static AmazonS3 createS3Client(
         new RetryPolicy(
             PredefinedRetryPolicies.DEFAULT_RETRY_CONDITION,
             PredefinedRetryPolicies.DEFAULT_BACKOFF_STRATEGY,
-            5,
```

Comment:
Can we get this parameter from a config?

Reply:
Probably. Should we test if this change will actually fix the tron jobs first? If yes, we can create another PR to get this param from a config file.

Reply:
Why not just have one PR which gets the parameter value externally? It's never a good practice to hard-code these values in code.